frontend/src/features/rollout-announcement/components/ProgressIndicator.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMemo } from 'react'
+import { Box } from '@chakra-ui/react'
+
+import { MotionBox } from '~components/motion'
+
+const ActiveIndicator = (): JSX.Element => (
+  <Box
+    width="1.5rem"
+    height="0.5rem"
+    borderRadius="full"
+    backgroundColor="secondary.500"
+  />
+)
+
+interface CircleIndicatorProps {
+  onClick: () => void
+  isActiveIndicator: boolean
+}
+
+const CircleIndicator = ({
+  onClick,
+  isActiveIndicator,
+}: CircleIndicatorProps): JSX.Element => {
+  return (
+    <Box
+      width="0.5rem"
+      height="0.5rem"
+      borderRadius="full"
+      backgroundColor="secondary.200"
+      marginRight={isActiveIndicator ? '1.5rem' : '0.5rem'}
+      onClick={onClick}
+      as="button"
```

Comment:
add aria-label for accessibility e.g. aria-label="Page X of Y". might want to ask @pearlyong for focus styles for the pagination dots.

Reply:
Okay I'll add aria-label and focus styles in a separate PR. I created an [issue](https://github.com/opengovsg/FormSG/issues/3977) to track this